lib/journald/src/lib.rs

```diff
@@ -0,0 +1,113 @@
+#![allow(non_camel_case_types)]
```

Comment:
I think it would be nice if we wrote a moduel doc here just to explain what this library does and why we needed it etc.

Reply:
Ok